keyboards/hineybush/h101/info.json

```
@@ -0,0 +1,381 @@
+{
+    "manufacturer": "hiney",
+    "keyboard_name": "h101",
+    "maintainer": "hineybush",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B0", "B1", "B2", "B10", "B11", "B12", "A8", "A9", "A10", "A15", "B3", "B4", "B5", "B6", "B7", "B9", "A3", "F0", "F1", "A7", "A5", "A4"],
+        "rows": ["C13", "C14", "C15", "A0", "A1", "A2"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "vid": "0x4069",
+        "pid": "0x0002"
+    },
+    "indicators": {
+        "num_lock": "A6",
+        "caps_lock": "B14",
+        "scroll_lock": "B8",
+        "on_state": 0
+    },
+    "layouts": {
+        "LAYOUT_h101_tsangan": {
```

Layout names shouldn't contain the keyboard name.
```suggestion
        "LAYOUT_tsangan": {
```